recipes-devtools/clang/clang_git.bb

```diff
@@ -74,10 +81,10 @@ EXTRA_OECMAKE="-DLLVM_ENABLE_RTTI=True \
 "
 
 EXTRA_OECMAKE_append_class-native = "\
-               -DLLVM_TARGETS_TO_BUILD='AArch64;ARM;Mips;PowerPC;X86' \
+               -DLLVM_TARGETS_TO_BUILD="${@get_clang_host_arch(bb, d)};${@get_clang_target_arch(bb, d)}" \
 "
 EXTRA_OECMAKE_append_class-nativesdk = "\
-               -DLLVM_TARGETS_TO_BUILD='AArch64;ARM;Mips;PowerPC;X86' \
+               -DLLVM_TARGETS_TO_BUILD="${@get_clang_host_arch(bb, d)};${@get_clang_target_arch(bb, d)}" \
```

Comment:
There is a advantage of building all backends once for all, since I have to test all architectures, it just means that I only build it once ( clang-native ) all other arches then can reuse it, so overall its a big saver for me. I understand that it may be a waste of time for someone who has to deal with limited set of targets.

Could you instead make the LLVM_TARGET_TO_BUILD such that it can be overridden either from config space or via bbapend ?

Then we can have a default value pointing to limited arches as default and I can override it with superset.